LabController/src/bkr/labcontroller/concurrency.py

```diff
@@ -32,6 +32,7 @@ def _read_from_pipe(f):
     discarding = False
     while True:
         try:
+            gevent.socket.wait_read(f.fileno())
```

Comment:
Add intention to the commit message. This is only required on Python 3.
The first read may be empty and we abort early.